src/main.ts

```diff
@@ -133,6 +135,19 @@ for await (const app of apps) {
         appRoles: app.appRoles,
       });
     }
+
+    if (
+      app.preferredSingleSignOnMode &&
+      app.preferredSingleSignOnMode == "saml"
+    ) {
+      await addSsoSamlUris({
```

Comment:
are these specific to saml?

https://learn.microsoft.com/en-us/entra/identity-platform/reference-app-manifest#identifieruris-attribute

seems to be general purpose, would it be better to not gate behind SAML and instead just do it based on identifier urls being set?